### PR TITLE
Jorge Review to Languages, and more

### PR DESCRIPTION
**Pull request by jorgedlcruz**
Here are all the changes I made for the Install Guides, system requirements, delete the Suse which is deprecated for so long, Languages, and much more.

### CODE REVIEW

File: installing.adoc [hunk 1]
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
 4. `Use {product-short}'s package repository [Y]` +
 displays, press `enter` to continue. Your system will be configured to add
 the {product-short} packaging repository for *yum* or *apt-get* as appropriate
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+*{product-short} IMAPD is meant to be installed on a multi-server environment, with
+hundred of thousands of users, and it's still as Beta. {product-short} recommends
```
Comment (tmclane) [inline installing.adoc]: as Beta = a Beta release

Comment (jorgedlcruz) [reply installing.adoc]: changed and added a new version

**Comment by gordyt** [comment]
@jorgedlcruz  I am verifying this with John but I think there may be some additional updates required, similar to the work that John already did here:

    bf714a6 [John Eastman] Cleanup in preparation for translations.
    Ordered lists have been indented per recommendation from po4a. 
    Indentation cleanup of existing lists.

Will let you know what I find out.

**Comment by jorgedlcruz** [comment]
Okay @gordyt please let me know, the edits were so extensive across the whole document. We are aiming to have this as /latest as discussed before. Let me know the edits we should add as well. Or maybe we can merge this and then do the additional changes after.

**Comment by gordyt** [comment]
Will do.  I believe that we will want to make these changes first.  If I understand the issue correctly, there are some problems generating the files required to do language translation without them.  

BTW, is OK for now, but for future PRs please base them off of the `develop` branch.  Note that I did do the whole `latest` thing yesterday (for admin and install guides), and is very easy to update after we merge your PR.

File: installing.adoc [hunk 2]
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
 4. `Use {product-short}'s package repository [Y]` +
 displays, press `enter` to continue. Your system will be configured to add
 the {product-short} packaging repository for *yum* or *apt-get* as appropriate
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+*{product-short} IMAPD is meant to be installed on a multi-server environment, with
```
Comment (gordyt) [inline installing.adoc]: @jorgedlcruz Here and anywhere else where you have bulleted or numbered list items, please be sure to have a single space between the marker and the text that follows.

File: packages.adoc [hunk 3]
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
...
+to the server extension and offer to the users a rich point-to-point text-chat conversations.
+*{product-short} Chat is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Chat package must be installed on the different *{product-short}-Store* server
+while having a multi-server environment.
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all
```
Comment (gordyt) [inline packages.adoc]: same here

File: packages.adoc [hunk 4]
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
...
+The {product-short} Chat package must be installed on the different *{product-short}-Store* server
+while having a multi-server environment.
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all 
+the necessary modules to connect and authenticate *{product-short} Users against a ownCloud or
+NextCloud Server, and an End-User Zimlet which allow users to perform actions to the their docu-
+ments stored on ownCloud or Nextcloud.
+*{product-short} Drive is marked as GA and supported starting {product-name} 8.8.
```
Comment (gordyt) [inline packages.adoc]: ditto

File: systemrequirements.adoc [hunk 5]
```diff
@@ -115,31 +116,26 @@ a|Administrator Computers +
  Other configurations may work.
 |The following operating system/browser combinations are supported:
 
-// TODO this looks well out of date.  How about Windows 10?  Edge browser?
-Windows XP with required updates, Vista, Windows 7, or Windows 8 with
-one of the following:
+*Windows Vista SP 2, Windows 7, Windows 8 or Windows 10 with one of the following:*
```
Comment (gordyt) [inline systemrequirements.adoc:119]: here

File: systemrequirements.adoc [hunk 6]
```diff
@@ -115,31 +116,26 @@ a|Administrator Computers +
...
-** Google Chrome
-
-Mac OS X 10.5, 10.6, 10.7, or 10.8 with one of the following:
+* Microsoft support is only available for Internet Explorer 11 or Microsoft Edge
+** IE11 and higher for Vista SP 2/Windows 7
+** IE11 and higher for Windows 8
+** IE11 or Microsoft Edge (Supported since ZCS 8.6 P4 and above) for Windows 10 
 
-* The latest stable release of:
+*The latest stable release of:
```
Comment (gordyt) [inline systemrequirements.adoc]: here

File: systemrequirements.adoc [hunk 7]
```diff
@@ -162,31 +158,26 @@ Recommended
 
 The following operating system/browser combinations are supported:
 
-Windows XP with required updates, Vista, Windows 7, Windows 8 or Windows 10 with one of the following:
+*Windows Vista SP 2, Windows 7, Windows 8 or Windows 10 with one of the following:*
```
Comment (gordyt) [inline systemrequirements.adoc]: here

File: systemrequirements.adoc [hunk 8]
```diff
@@ -162,31 +158,26 @@ Recommended
...
+* Microsoft support is only available for Internet Explorer 11 or Microsoft Edge
+** IE11 and higher for Vista SP 2/Windows 7
+** IE11 and higher for Windows 8
+** IE11 or Microsoft Edge (Supported since ZCS 8.6 P4 and above) for Windows 10 
 
-// TODO El Capitan is 10.11 and that isn't even the latest...
-Mac OS X 10.5, 10.6, 10.7, or 10.8 with one of the following:
-
-* The latest stable release of:
+*The latest stable release of:
```
Comment (gordyt) [inline systemrequirements.adoc]: here

**Review by gordyt — COMMENTED** [review]
@jorgedlcruz I've noted the places that need that minor update for compatibility with the translation tools.

File: systemrequirements.adoc (hunk 5 above)
Comment (jorgedlcruz) [reply systemrequirements.adoc:119]: This is bold not bullet

File: systemrequirements.adoc (hunk 6 above)
Comment (jorgedlcruz) [reply systemrequirements.adoc]: fixed

File: systemrequirements.adoc (hunk 7 above)
Comment (jorgedlcruz) [reply systemrequirements.adoc]: This is bold not bullet

File: systemrequirements.adoc (hunk 8 above)
Comment (jorgedlcruz) [reply systemrequirements.adoc]: fixed

File: packages.adoc (hunk 4 above)
Comment (jorgedlcruz) [reply packages.adoc]: fixed, it didn't need bold

File: packages.adoc (hunk 3 above)
Comment (jorgedlcruz) [reply packages.adoc]: fixed, it didn't need bold

File: installing.adoc (hunk 2 above)
Comment (jorgedlcruz) [reply installing.adoc]: fixed, it didn't need bold or bullet

**Comment by gordyt** [comment]
@jorgedlcruz let me go ahead and generate a test export so folks can see the final product.
One sec...

**Comment by jorgedlcruz** [comment]
Appreciated @gordyt thank you as usual for all the hard work!

**Comment by gordyt** [comment]
@jorgedlcruz  I had to push in one fix to your branch for this:

    $ asciidoctor -b html5 -o single.html index-single-server-install.adoc
    asciidoctor: WARNING: installing.adoc: line 330: list item index: expected 15, got 16
    asciidoctor: WARNING: installing.adoc: line 338: list item index: expected 16, got 17
    asciidoctor: WARNING: installing.adoc: line 349: list item index: expected 17, got 18

Generated a `gh-pages` export with the results.  You can view them here:

* [single-server](https://zimbra.github.io/installguides/work-in-progress/single.html)
* [multi-server](https://zimbra.github.io/installguides/work-in-progress/multi.html)
* [upgrade](https://zimbra.github.io/installguides/work-in-progress/upgrade.html)

Please go through the results and check the formatting.

File: installing.adoc [hunk 9]
```diff
@@ -96,59 +103,70 @@ Found zimbra-store (local)
...
+Install zimbra-archiving [N] y
 
-Install zimbra-archiving [N] N
+Install zimbra-chat [Y] y
 
-Install zimbra-chat [Y] N
+Install zimbra-drive [Y] y
 
-Install zimbra-drive [Y] N
+Install zimbra-imapd [Y] n
```
Comment (plobbes) [inline installing.adoc]: nit: trailing whitespace

**Comment by gordyt** [comment]
@jorgedlcruz I'm adding @gaylebillat as a content reviewer.

File: systemrequirements.adoc [hunk 10]
```diff
@@ -115,31 +116,26 @@ a|Administrator Computers +
...
-Mac OS X 10.5, 10.6, 10.7, or 10.8 with one of the following:
+* Microsoft support is only available for Internet Explorer 11 or Microsoft Edge
+** IE11 and higher for Vista SP 2/Windows 7
+** IE11 and higher for Windows 8
+** IE11 or Microsoft Edge (Supported since ZCS 8.6 P4 and above) for Windows 10 
 
 * The latest stable release of:
 ** Firefox
-** Safari
+** Safari
```
Comment (plobbes) [inline systemrequirements.adoc]: nit: trailing whitespace

File: systemrequirements.adoc [hunk 11]
```diff
@@ -162,31 +158,26 @@ Recommended
...
```
Comment (plobbes) [inline systemrequirements.adoc]: nit: trailing whitespace

**Review by plobbes — COMMENTED** [review]
whitespace nits

File: installconfig.adoc [hunk 12]
```diff
@@ -79,6 +79,17 @@ endif::networkeditiondoc[]
 | Select the sub-menu for
   <<imapd_config,IMAPD configuration>>
 
+| zimbra-chat
+| Installing the Zimbra-Chat package is optional.
+  If you choose to install Zimbra-Chat for point-to-point text chat, it should
+  be installed on every Zimbra Store Server that is part of the Zimbra configuration.
+
+| zimbra-drive
+| Installing the Zimbra-Chat package is optional.
```
Comment (fatkudu) [inline installconfig.adoc]: s/Zimbra-Chat/Zimbra/Drive/

File: installconfig.adoc [hunk 13]
```diff
@@ -79,6 +79,17 @@ endif::networkeditiondoc[]
...
+  If you choose to install Zimbra-Chat for point-to-point text chat, it should
```
Comment (fatkudu) [inline installconfig.adoc]: Is this true for Zimbra-Drive true?  If so, update accordingly or replace with appropriate text.

File: installing.adoc [hunk 14]
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
 4. `Use {product-short}'s package repository [Y]` +
 displays, press `enter` to continue. Your system will be configured to add
 the {product-short} packaging repository for *yum* or *apt-get* as appropriate
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+{product-short} IMAPD is meant to be installed on a multi-server environment, with
+hundred of thousands of users, and it's still a Beta Release. {product-short} recommends
```
Comment (fatkudu) [inline installing.adoc]: s/hundred/hundreds/

File: installing.adoc [hunk 15]
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
...
+to only install it for demo and evaluation purposes.*
+* If installing *{product-short} Network Edition*, you will have the modules Archiving
+and Convertd as an option to add to your {product-short} installation.
+* {product-short}-dnscache is recommended to be installed on the MTA Servers only when
```
Comment (fatkudu) [inline installing.adoc]: It is recommended to only install *{product-short}-dnscache* on the MTA Servers  when there are no other DNS servers (for example *dnsmasq* or *bind*) already installed and running on the same server.

File: installing.adoc [hunk 16]
```diff
@@ -96,59 +103,70 @@ Found zimbra-store (local)
...
-Install zimbra-imapd [Y] Y
+Install zimbra-network-modules-ng [Y] y
 
-.
-.
+###WARNING###
+
+Network Modules NG needs to bind on TCP ports 8735 and 8736 in order
+to operate, for inter-instance communication.
+Please verify no other service listens on these ports and that
```
Comment (fatkudu) [inline installing.adoc:155]: Please verify that no other service...

File: packages.adoc [hunk 17]
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
 contact {product-short} sales.
 endif::networkeditiondoc[]
 
+* *{product-short} Chat:* Installing this package is optional. This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Chat includes a server extension with all
```
Comment (fatkudu) [inline packages.adoc]: Missing '*' after Chat

File: packages.adoc [hunk 18]
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
...
 endif::networkeditiondoc[]
 
+* *{product-short} Chat:* Installing this package is optional. This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Chat includes a server extension with all 
+the necessary modules to run an embedded XMPP Server, and an End-User Zimlet which connects
+to the server extension and offer to the users a rich point-to-point text-chat conversations.
+{product-short} Chat is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Chat package must be installed on the different *{product-short}-Store* server
```
Comment (fatkudu) [inline packages.adoc]: I don't understand the meaning of this sentence - sorry.

File: packages.adoc [hunk 19]
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
...
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all 
+the necessary modules to connect and authenticate *{product-short} Users against a ownCloud or
+NextCloud Server, and an End-User Zimlet which allow users to perform actions to the their docu-
+ments stored on ownCloud or Nextcloud.
+{product-short} Drive is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Drive package must be installed on the different *{product-short}-Store* server
```
Comment (fatkudu) [inline packages.adoc]: Again, don't actually know what this means.  Does Drive need to be on its own server?  How does that work?

File: packages.adoc [hunk 20]
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
...
+while having a multi-server environment.
++
+
+[WARNING]
+Zimbra Drive provides only a connectivity to a ownCloud or NextCloud Server. And is the Customer
+responsibility to maintain, backup, and protect the data stored on this OC or NC Servers.
```
Comment (fatkudu) [inline packages.adoc]: s/on this OC or NC Servers/on these ownCloud or NextCloud Servers/

**Review by fatkudu — CHANGES_REQUESTED** [review]

File: installing.adoc (hunk 16 above)
Comment (jorgedlcruz) [reply installing.adoc:155]: This is a comment on the own product installation, I will not edit it on the documentation.

File: installing.adoc (hunk 15 above)
Comment (jorgedlcruz) [reply installing.adoc]: done

File: installing.adoc (hunk 14 above)
Comment (jorgedlcruz) [reply installing.adoc]: done

File: installconfig.adoc (hunk 13 above)
Comment (jorgedlcruz) [reply installconfig.adoc]: done

File: installconfig.adoc (hunk 12 above)
Comment (jorgedlcruz) [reply installconfig.adoc]: done

File: packages.adoc (hunk 17 above)
Comment (jorgedlcruz) [reply packages.adoc]: done

File: packages.adoc (hunk 18 above)
Comment (jorgedlcruz) [reply packages.adoc]: edited

File: packages.adoc (hunk 19 above)
Comment (jorgedlcruz) [reply packages.adoc]: The Drive and Chat package for Zimbra consist on a server extension and a zimlet, so the Drive or Chat package must be installed on all Zimbra-Stores on the Infrastructure. Or at least on the ones where we want the users to be able to access Drive/Chat. I've edited, take a look now.

File: packages.adoc (hunk 20 above)
Comment (jorgedlcruz) [reply packages.adoc]: done

File: systemrequirements.adoc (hunk 10 above)
Comment (jorgedlcruz) [reply systemrequirements.adoc]: done

File: systemrequirements.adoc (hunk 11 above)
Comment (jorgedlcruz) [reply systemrequirements.adoc]: done

File: packages.adoc [hunk 21]
```diff
@@ -109,6 +109,32 @@ license usage. To find out more about {product-short} Archiving and Discovery,
...
+the necessary modules to run an embedded XMPP Server, and an End-User Zimlet which connects
+to the server extension and offer to the users a rich point-to-point text-chat conversations.
+{product-short} Chat is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Chat package must be selected and installed on every *{product-short}-Store*.
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all
```
Comment (fatkudu) [inline packages.adoc:123]: Looks like a couple of places here and on the next line where BOLD is started but not finished?

**Review by fatkudu — CHANGES_REQUESTED** [review]
Looking good.  Only 1 minor query about bolding.

**Comment by jorgedlcruz** [comment]
So now with all the suggested changes, @gordyt do you think you can generate another HTML, which I hope is the final version, for now at least?

**Comment by gordyt** [comment]
@jorgedlcruz will do.  Gotta finish a few things first.  Will post update here when live.

**Comment by gordyt** [comment]
@jorgedlcruz I have redone the export and went ahead and published it to the `8.8.3` (`latest`) location.

* [single-server](https://zimbra.github.io/installguides/latest/single.html)
* [multi-server](https://zimbra.github.io/installguides/latest/multi.html)
* [upgrade](https://zimbra.github.io/installguides/latest/upgrade.html)

*If* you do have to make any additional changes, please be sure and update your local `jorge-review` branch with the latest copy of it from `origin`.  You had inadvertently done a merge from `origin/jorge-review` to your local `jorge-review` (rather than rebasing your local `jorge-review` onto `origin/jorge-review`) and ended up with a wonky merge commit. 

For example, you would do the following from your local checkout:

    git fetch origin
    git checkout -B jorge-review origin/jorge-review

That will reset your *local* branch to be correctly in sync with `origin`.

**Pull request closed without merging**